<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CdnPurger</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1,5,1,8,1">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>네이버 클라우드 CDN Purger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="settingGroupBox">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="cdnInstanceNo"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_cdnInstanceNo">
           <property name="text">
            <string>cdnInstanceNo : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_isWholeDomain">
           <property name="text">
            <string>isWholeDomain : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <item>
            <widget class="QRadioButton" name="radioButton_isWholeDomain_Yes">
             <property name="text">
              <string>Yes</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_IsWholeDomain</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_isWholeDomain_No">
             <property name="text">
              <string>No</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_IsWholeDomain</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_isWholePurge">
           <property name="text">
            <string>isWholePurge : </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="radioButton_isWholePurge_Whole">
             <property name="text">
              <string>Whole</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_isWholePurge</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_isWholePurge_Directory">
             <property name="text">
              <string>Directory</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_isWholePurge</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_isWholePurge_Files">
             <property name="text">
              <string>Files</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_isWholePurge</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_domainEdit">
      <property name="text">
       <string>• Purge할 도메인명을 입력하세요.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="domainEdit">
      <property name="placeholderText">
       <string>여러 도메인 작업 시 줄을 바꿔 입력합니다.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pathEdit">
      <property name="text">
       <string>• Purge할 파일 경로명을 입력하세요. (예) /sample/sample.jpg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pathEdit">
      <property name="placeholderText">
       <string>여러 파일 작업 시 줄을 바꿔 입력합니다.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="submitButtonLayout" stretch="0,0">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuVersion">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="action_keySetting"/>
    <addaction name="action_clearSetting"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="action_minimize"/>
    <addaction name="action_maximize"/>
    <addaction name="action_normal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuWindow"/>
   <addaction name="menuVersion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_keySetting">
   <property name="text">
    <string>Key Setting</string>
   </property>
   <property name="toolTip">
    <string>Key Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_minimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="action_maximize">
   <property name="text">
    <string>Maximize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="action_normal">
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_clearSetting">
   <property name="text">
    <string>Clear Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_isWholePurge"/>
  <buttongroup name="buttonGroup_IsWholeDomain"/>
 </buttongroups>
</ui>
